content/python/concepts/requests-module/requests-module.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Requests'
+Description: 'The requests module is the de facto standard for sending HTTP requests in Python.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Modules'
+  - 'APIs'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`requests`** module is the de facto standard for sending HTTP requests in Python.
+
+It abstracts the complexities of making requests behind a simple API so that code can focus on interacting with services and consuming data in the application.
```

Comment:
```suggestion
It abstracts the complexities of making requests behind an [API](https://www.codecademy.com/resources/docs/general/api) so that code can focus on interacting with services and consuming data in the application.
```